### PR TITLE
Refactor the telco5g CI e2e periodic job

### PR DESCRIPTION
**Pull request by sshnaidm**

### CODE REVIEW

File: ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+else
+# Run on downstream cnfdc1 without bastion
+cat << EOF > $SHARED_DIR/inventory
+[hypervisor]
+${DSHVIP} ansible_host=${DSHVIP} ansible_ssh_user=kni ansible_ssh_common_args="${COMMON_SSH_ARGS}" ansible_ssh_private_key_file="${SSH_PKEY}"
+EOF
+
+fi
+echo "#############################################################################..."
+echo "========  Deploying plan $PLAN_NAME on cluster $CLUSTER_NAME with$(if ! $BASTION_ENV; then echo "out"; fi) a bastion  ========"
```
Comment (sabinaaledort) [inline ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh:105]: Can we change the if so it only prints "with bastion"? "without bastion" might be confusing.

Comment (sshnaidm) [reply ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh:105]: fixing in https://github.com/openshift/release/pull/32537

File: ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh [hunk 2]
```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ telco cluster setup command ************"
+# TODO: Remove once OpenShift CI will be upgraded to 4.2 (see https://access.redhat.com/articles/4859371)
```
[the remark this answers is not in the document]
Comment (sshnaidm) [reply ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh:8]: fixing in https://github.com/openshift/release/pull/32537

**Pull request merged**